src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -89,12 +92,30 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   // Toggle
   --pf-c-data-toolbar__toggle--m-expanded__c-button--m-plain--Color: var(--pf-global--Color--100);
 
+  // Divider
+  --pf-c-data-toolbar--c-divider--m-vertical--spacer: var(--pf-c-data-toolbar--spacer--base);
+
   position: relative;
   row-gap: var(--pf-c-data-toolbar--RowGap);
   display: grid;
   padding-top: var(--pf-c-data-toolbar--PaddingTop);
   padding-bottom: var(--pf-c-data-toolbar--PaddingBottom);
   background-color: var(--pf-c-data-toolbar--BackgroundColor);
+
+  // set this var here so specificity is 20
+  .pf-c-divider {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--c-divider--m-vertical--spacer);
+  }
+
+  .pf-c-divider.pf-m-vertical {
```

Comment:
Since these styles aren't scoped, they'll apply to any use of the vertical divider in the component. I think it's fair to assume we or someone might use a divider as part of another component, say in a dropdown item or toggle or something like that. Can we scope the styles somehow? It would be fair to keep the separator container to manage the divider positioning, then use the divider in that, similar to how we do that in other components to handle the positioning of a component in a component.

Reply:
Scoped 👍